build.gradle

```diff
@@ -33,6 +33,9 @@ buildscript {
             throw new IllegalArgumentException("'kotlin_snapshot_version' should be defined when building with snapshot compiler")
         }
     }
+    ext.jvm_ir_enabled = rootProject.properties['enable_jvm_ir'] != null
+    ext.jvm_ir_api_check_enabled = rootProject.properties['enable_jvm_ir_api_check'] != null
+    ext.native_targets_enabled = rootProject.properties['disable_native_targets'] == null
 
```

Comment:
Why are you using `-Xallow-jvm-ir-dependencies` in atomicfu, but not here?

Reply:
This flag was was used in atomicfu only for test project for testing gradle plugin. Here we will fairly use IR dependency with IR compiled project